substation-diagram/substation-diagram-core/src/test/resources/TestSubstation.svg

```diff
@@ -40,20 +69,20 @@
 <text x="-5.0" font-size="8" y="-5.0" transform="rotate(0,0,0)" font-family="Verdana" class="component-label" id="l_N_LABEL">l</text>
         </g>
         <g class="substation-diagram DISCONNECTOR d" id="d" transform="translate(91.0,356.0)">
-    <g class="closed" id="closed">
+    <g class="closed" id="DISCONNECTOR-closed">
```

Comment:
Yes, but maybe we should create another issue for that, because it's not directly related to this evolution, and there are other component svg defining the "id" attribute and concerned by the same issue (2WT, 3WT, SVC ...)

Reply:
Ok let's fix it in another issue. Could you sync with the master?

Reply:
Conflicts are resolved : you can merge now ...